index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Merriweather&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <img class="logo" src="./images/environnews-logo.png" alt="environnews-logo">
+    <nav class="big-screen-nav">
+      <a href="#">LATEST NEWS</a>
+      <a href="#">WILDLIFE</a>
+      <a href="#">GLOBAL WARMING</a>
+    </nav>
+  </header>
+
+  <main>
+    <div class="big-news">
+      <img src="./images/fruit-peel.jpg" alt="fruit peel image">
+      <div class="big-news-text">
+        <h1>Scientists Use Fruit Peel to Turn Old Batteries Into New</h1>
+        <p>Scientists have developed a novel method of using fruit peel waste to extract and reuse precious metals from
+          spent lithium-ion batteries in order to create new batteries. The scientists say that their waste-to-resource
+          approach tackles both food waste and electronics waste, supporting the development of a circular economy with
+          zero waste.</p>
+      </div>
+    </div>
+
+    <div class="other-news">
+      <div class="other-news-card">
+        <img src="./images/lead.jpg" alt="pencil image">
+        <h1>New Device Can Measure Toxic Lead Within Minutes</h1>
+      </div>
+
+      <div class="other-news-card">
+        <img src="./images/ozone.jpg" alt="sky image">
+        <h1>Experts Reveal Major Holes in International Ozone Treaty</h1>
+      </div>
+
+      <div class="other-news-card">
+        <img src="./images/rice.jpg" alt="rice field image">
+        <h1>Antagonistic Genes Modify Rice Plant Growth</h1>
+      </div>
+
+      <div class="other-news-card">
+        <img src="./images/ice.jpg" alt="ice image">
+        <h1>Atmospheric Scientists Study Fires to Resolve Ice Question in Climate Models</h1>
+      </div>
+      <div class="other-news-card">
```

Comment:
Be consistent with line breaks between the cards, it makes it easier to read your code :)